release-note.md

```diff
@@ -1,5 +1,15 @@
 # Release Note
 
+## 5.3.1
+
+Released on 8/11/2019
+
+This is a patch release for fixing the Google BigQuery data source issue.
```

Comment:
we should make it something like below. 

"There was an issue that some of our users had a problem for importing data from Google BigQuery due to the underlying Google BigQuery API change. We have updated all the related places to use the officially supported APIs."